<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FpgaDialog</class>
 <widget class="QDialog" name="FpgaDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gmOptGroup">
     <property name="title">
      <string>GM Settings</string>
     </property>
     <widget class="QWidget" name="layoutWidget13">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>20</y>
        <width>578</width>
        <height>640</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_14" columnstretch="30,0,0">
       <item row="5" column="0">
        <widget class="QLabel" name="delayTimeLabel">
         <property name="text">
          <string>Delay Time :</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="14" column="2">
        <widget class="QRadioButton" name="oneKHzRadio">
         <property name="text">
          <string>1KHz</string>
         </property>
        </widget>
       </item>
       <item row="15" column="1">
        <widget class="QRadioButton" name="tenKHz">
         <property name="text">
          <string>10KHz</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QComboBox" name="ADC2ChannelCombo">
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="delayTimeSpinner">
         <property name="maximum">
          <number>5100</number>
         </property>
         <property name="singleStep">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item row="16" column="1">
        <widget class="QCheckBox" name="anodePulserCheck">
         <property name="text">
          <string>Anode Pulser</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QRadioButton" name="unipolarRadio">
         <property name="text">
          <string>Unipolar</string>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QRadioButton" name="bipolarRadio">
         <property name="text">
          <string>Bipolar</string>
         </property>
        </widget>
       </item>
       <item row="16" column="0">
        <widget class="QLabel" name="pulserOptLabel">
         <property name="text">
          <string>Pulser Options :</string>
         </property>
        </widget>
       </item>
       <item row="17" column="0">
        <widget class="QLabel" name="NoOfPulsesLabel">
         <property name="text">
          <string>No of Pulses :</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="cathodeModeLabel">
         <property name="text">
          <string>Cathode Mode :</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="ADC1ChannelLbl">
         <property name="text">
          <string>ADC1 Channel :</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="14" column="1">
        <widget class="QRadioButton" name="hundredHzRadio">
         <property name="text">
          <string>100Hz</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="ADC2ChannelLabel">
         <property name="text">
          <string>ADC2 Channel :</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="20" column="2">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSpinBox" name="timestampResSpinner">
         <property name="maximum">
          <number>25500</number>
         </property>
         <property name="singleStep">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="17" column="1">
        <widget class="QComboBox" name="noOfPulsesCombo">
         <item>
          <property name="text">
           <string>0</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>11</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>13</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>14</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>15</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="GMChannel1TestModeCheck">
         <property name="text">
          <string>Channel 1 Test Mode</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="timestampResLabel">
         <property name="text">
          <string>Timestamp Resolution :</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QCheckBox" name="timeDetectCheck">
         <property name="text">
          <string>Time Detect Readout</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QCheckBox" name="negEnergyCheck">
         <property name="text">
          <string>Negative Energy Readout</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="GMDisPhotonCollectCheck">
         <property name="text">
          <string>Disable Photon Collect</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QComboBox" name="ADC1ChannelCombo">
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="GMPixelMapCheck">
         <property name="text">
          <string>Enable Pixel Mapping</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="gmReadoutOptLab">
         <property name="text">
          <string>Readout Options :</string>
         </property>
        </widget>
       </item>
       <item row="18" column="0" colspan="3">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="pulserFreqLabel">
         <property name="text">
          <string>Pulser Frequency :</string>
         </property>
        </widget>
       </item>
       <item row="11" column="2">
        <widget class="QRadioButton" name="sparsifiedRadio">
         <property name="text">
          <string>Sparsified</string>
         </property>
        </widget>
       </item>
       <item row="12" column="2">
        <widget class="QRadioButton" name="multiThreshRadio">
         <property name="text">
          <string>Multi Threshold</string>
         </property>
        </widget>
       </item>
       <item row="15" column="2">
        <widget class="QRadioButton" name="hundredKHz">
         <property name="text">
          <string>100KHz</string>
         </property>
        </widget>
       </item>
       <item row="16" column="2">
        <widget class="QCheckBox" name="cathodePulserCheck">
         <property name="text">
          <string>Cathode Pulser</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="defaultGMLab">
         <property name="text">
          <string>Default GM :</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="19" column="1" colspan="2">
        <widget class="QPushButton" name="UpdateGMButton">
         <property name="text">
          <string>Update GM</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="GMDebugCheck">
         <property name="text">
          <string>Debug</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="9" column="1">
        <widget class="QComboBox" name="defaultGMCombo">
         <item>
          <property name="text">
           <string>-</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="readoutModeLabel">
         <property name="text">
          <string>Readout Mode:</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QRadioButton" name="readAllRadio">
         <property name="text">
          <string>ReadAll</string>
         </property>
        </widget>
       </item>
       <item row="20" column="1">
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
